core/src/components/modal/modal.tsx

```diff
@@ -560,6 +582,43 @@ export class Modal implements ComponentInterface, OverlayInterface {
     return eventMethod(this.el, 'ionModalWillDismiss');
   }
 
+  /**
+   * Move a sheet style modal to a specific breakpoint.
+   */
+  @Method()
+  async setBreakpoint(breakpoint: number): Promise<void> {
```

Comment:
~ What if we made this function return `Promise<boolean>`? Returning `false` when the value is not set and returning true when it completes successfully?

Reply:
For what case would this help? 🤔 i don't know if this is important to know in a running application, i guess this will only occour in active development and the warnings should be enough then?

Reply:
My thought was around at the time of calling the function, being able to know if it actually changed the breakpoint value or not. 

Since `setBreakpoint` will skip/do nothing if the breakpoint matches the current breakpoint, there's no way for a developer to wire logic that is dependent on cases where it did nothing. 

I think this isn't required though. With the addition of the new events, developers can minimally wire logic that responds to when the breakpoint value is changed.